### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by HaSistrunk**
Fixes issue #20

### CODE REVIEW

**Comment by helrond** [comment]
This should give you what you need: https://github.com/RockefellerArchiveCenter/pictor/blob/development/pictor/urls.py

**Review by helrond — APPROVED** [review]
for the record I'm fine with this being merged even if the build is failing (thanks for nothing Travis)

File: README.md [hunk 1]
```diff
@@ -1,23 +1,75 @@
...
+pictor configurations are stored in `/pictor/config.py`. This file is excluded from version control, and you will need to update this file with values for your local instance.
+
+The first time the container is started, the example config file (`/pictor/config.py.example`) will be copied to create the config file if it doesn't already exist.
+## Requirements
+
+Using this repo requires having [Docker](https://store.docker.com/search?type=edition&offering=community) installed.
+
+## Services
+pictor receives content from an external service. It expects TIFF files that are in bags that
+contain their ArchivesSpace RefID in the bag-info.txt file. For an example of what pictor expects to receive, see the `fixtures/` directory.
```
Comment (bonniegee) [inline README.md]:
```suggestion
contain their ArchivesSpace URI in the bag-info.txt file. For an example of what pictor expects to receive, see the `fixtures/` directory.
```

**Review by bonniegee — CHANGES_REQUESTED** [review]
One small change--otherwise looks great!

**Review by p-galligan — APPROVED** [review]
Looks good to me!

**Review by bonniegee — APPROVED** [review]

**Pull request merged**